usr/src/uts/armv8/os/mp_startup.c

```diff
@@ -676,8 +654,10 @@ start_cpu(processorid_t who, uint64_t ignore_affinity)
 }
 
 void
-start_other_cpus(int cprboot)
+start_other_cpus(int flag)
 {
+	_NOTE(ARGUNUSED(flag));
```

Comment:
This is an old way of saying this for lint(1) which we don't use anymore, you can just put `__unused` after `flag` in the argument list